### PR TITLE
[#47] Fix that makes the manager work for write db correctly

### PR DESCRIPTION
**Pull request by vikasgulati**
- works with using
- works with routers configured for read/write scenario

### CODE REVIEW

File: bulk_update/manager.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-    def bulk_update(self, objs, update_fields=None,
-                    exclude_fields=None, batch_size=None):
-        return bulk_update(
-            objs, update_fields=update_fields,
-            exclude_fields=exclude_fields, using=self.db,
-            batch_size=batch_size)
+class BulkUpdateQuerySet(models.QuerySet):
+    def bulk_update(self, objs, update_fields=None, exclude_fields=None):
+        self._for_write = True
+        bulk_update(objs, update_fields=update_fields,
```
Comment (aykut) [inline bulk_update/manager.py]: I think you forgot to add `batch_size` parameter.

**Comment by aykut** [comment]
Hey, thanks for the PR.

Can you please fix test errors. Thanks.

File: bulk_update/manager.py (hunk 1 above)
Comment (vikasgulati) [inline bulk_update/manager.py]: yup my bad. Added it.

**Comment by vikasgulati** [comment]
@aykut: As I see test case errors were version dependent issues. A bit caught up with something right now. Would fix it in some time.

**Comment by aykut** [comment]
Thanks a lot.

**Pull request closed without merging**